test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -347,7 +347,7 @@ public boolean getDeploymentConfigStatus(String deploymentConfigName) {
     }
 
     public Secret createSecret(Secret secret) {
-        return client.secrets().inNamespace(getNamespace()).create(secret);
+        return client.secrets().inNamespace(getNamespace()).createOrReplace(secret);
```

Comment:
Hmmm, why do we need this operation `createOrReplace` instead of `create`? Shouldn't we be warned that if we in test case create one secret with name `secret-example` and if we try create again same secret we should get error or?